### PR TITLE
Inconsequential tweaks from studying `tempchecker.py`

### PR DESCRIPTION
**Pull request by rebeccacremona**
While investigating (non-fatal) error messages thrown during session cleanup yesterday, I started adding comments for myself to `tempchecker.py`, and started making try/excepts narrower to make sure I understood which errors were being thrown where.

These commits are the results of my internal note-taking/experimenting. I thought I'd offer it as a PR in case it might be helpful to others.

Please feel free to simply close this entire PR: no pressure to accept any of it, much less all of it.

### CODE REVIEW

File: webrecorder/webrecorder/rec/tempchecker.py [hunk 1]
```diff
@@ -46,83 +57,91 @@ def delete_if_expired(self, temp_user, temp_dir):
...
+            # their collections will be migrated to storage.
+            # Clean up if that migration is complete (i.e. the dir is empty).
+            # Otherwise, wait.
+            if os.path.isdir(temp_dir):
+                try:
+                    logger.debug('TempChecker: Removing if empty: ' + temp_dir)
+                    os.rmdir(temp_dir)
+                    logger.debug('TempChecker: Deleted empty dir: ' + temp_dir)
+                except OSError as e:
+                    if e.errno == 90:
```
Comment (ikreymer) [inline webrecorder/webrecorder/rec/tempchecker.py]: Wondering if this should be errno 39 for ENOTEMPTY? Curious how 90 happened here, which is ENAMETOOLONG?

**Comment by ikreymer** [comment]
Thanks for adding this! Just had one question on the error code, and I think it should be good to merge.. definitely helps clear things up, eg. I forgot that 'temps to remove' is actually all temp users to examine! :)

File: webrecorder/webrecorder/rec/tempchecker.py (hunk 1 above)
Comment (rebeccacremona) [reply webrecorder/webrecorder/rec/tempchecker.py]: Wow: I really cannot reconstruct how I got `e.errno == 90`, here or anywhere else. How bizarre, I'm usually so methodical...... closing until I can figure out what I did; will reopen with a correction shortly....

**Comment by rebeccacremona** [comment]
Okay, still no idea how I bizarrely ended up with 90 there... replacing with semantic code references like `errno.ENOTEMPTY`. Thanks for catching! Would have been way embarrassing to break WR in this manner....

**Pull request merged**